lib/src/Base/Graph/DrawableImplementation.cxx

```diff
@@ -102,6 +102,7 @@ void DrawableImplementation::InitializeValidParameterList()
   ValidPointStyles.add("None");
 
   /* Accepted colors */
+  ColorCodes["transparent"]          = "#FFFFFF00";
```

Comment:
???

Reply:
all colors are opaque isnt it ? I needed a transparent Polygon fill color to only draw edges